mkt/site/tests/__init__.py

```diff
@@ -430,6 +430,12 @@ def shortDescription(self):
     def _pre_setup(self):
         super(TestCase, self)._pre_setup()
 
+        # XXX See if we can remove this when we switch to Django 1.8.
+        # Some crud from the migration system gets stuck here and we have to
+        # kick it loose manually.
+        from mkt.webapps.models import AddonUser
+        UserProfile.addonuser_set.related.model = AddonUser
```

Comment:
Bleeds into our tests because the migrations are run in the same process. If I run the tests with `REUSE_DB=1` they pass just fine. Why this FK? Not sure. I thought it was due to `UserForeignKey` but I replaced it with a regular `ForeignKey` and the problem persisted.